Makefile

```diff
@@ -316,7 +316,7 @@ printf "$$rows" "Postgres DB" "http://$(get_my_ip).nip.io:18080/?pgsql=postgres&
 printf "$$rows" "Portainer" "http://$(get_my_ip).nip.io:9000" admin adminadmin;\
 printf "$$rows" "Redis" "http://$(get_my_ip).nip.io:18081";\
 printf "$$rows" "Dask Dashboard" "http://$(get_my_ip).nip.io:8787";\
-printf "$$rows" "Docker Registry" "$${REGISTRY_URL}" $${REGISTRY_USER} $${REGISTRY_PW};\
+printf "$$rows" "Docker Registry" "http://$${REGISTRY_URL}/v2/_catalog" $${REGISTRY_USER} $${REGISTRY_PW};\
```

Comment:
a URL contains already http, is this not strange?

Reply:
Yes, it is indeed strange, but for the moment **it does not** 

For instance see
https://github.com/itisfoundation/osparc-simcore/blob/a3f559174aaaf099365d9b73116cbf1837ae24db/.env-devel#L160

and

https://github.com/itisfoundation/osparc-simcore/blob/ad654dc8038af2957e8658d00f831a84d80d7479/Makefile#L650

Reply:
Created issue https://github.com/ITISFoundation/osparc-simcore/issues/6055